tasklist/client/src/modules/components/BPMNDiagram/DiagramControls/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {IconButton} from '@carbon/react';
+import {Add, CenterCircle, Subtract} from '@carbon/react/icons';
```

Comment:
You go the same icons from Operate, right?

Reply:
These are the same icons from operate